user_management/api/tests/test_serializers.py

```diff
@@ -75,6 +75,7 @@ def test_deserialize_invalid_new_password(self):
         })
         self.assertFalse(serializer.is_valid())
         self.assertIn('new_password', serializer.errors)
+        self.assertNotIn('Your passwords do not match', serializer.errors)
```

Comment:
`Your passwords do not match` is a value, not a key.

Also, shouldn't it end in a `.`?